cpp_code/tests/test_random_number_generator.cpp

```diff
@@ -198,7 +228,7 @@ static bool psi_test(const vector<size_t> &counts,
 //      `Summary of operational information' on p. 602.
 
 static const size_t SHAPIRO_WILK_DF = 50;
-static const double SHAPIRO_WILK_CRITICAL = .930;  // At alpha=0.001.
+static const double SHAPIRO_WILK_CRITICAL = .930;  // alpha = .01
```

Comment:
I thought this alpha was documenting the value, not the test. That's worth documenting, but it would be unclear to document it here. Or was the significance of the critical value in error before?